.changeset/tough-dolphins-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-gcp': patch
```

Comment:
minor change here as it's the first release

Reply:
fixed